sdk/mediaservices/Azure.Media.Analytics.Edge/src/Customization/MediaGraphInstanceSetRequest.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Media.Analytics.Edge.Models
+{
+    public partial class MediaGraphInstanceSetRequest
+    {
+        internal MediaGraphInstanceSetRequestBody InstanceBody;
+
+        /// <summary>
+        ///  Serialize .
+        /// </summary>
+        /// <returns></returns>
+        public override string GetPayloadAsJson()
+        {
+            InstanceBody = new MediaGraphInstanceSetRequestBody(Instance.Name);
+            InstanceBody.SystemData = Instance.SystemData;
+            InstanceBody.Properties = Instance.Properties;
+            return InstanceBody.GetPayloadAsJson();
```

Comment:
Wouldn't this cause issues in a multithreaded scenario?

Reply:
Models are not intended to be thread safe.